src/smt/smt_engine.h

```diff
@@ -932,6 +951,18 @@ class CVC4_PUBLIC SmtEngine
    * unsatisfiable. If not, then the found solutions are wrong.
    */
   void checkSynthSolution();
+
+  /**
+   * Check that a solution to an interpolation conjecture is indeed a solution.
```

Comment:
Use "interpolation problem" rather than "interpolation conjecture" since you are using "conjecture" to refer to the `B` in the `get-interpolant` call.